jellyfin_kodi/downloader.py

```diff
@@ -282,6 +282,17 @@ def get_query_params(params, start, count):
             query['params'] = params
 
             result = result or {'Items': []}
+
+            # Mitigates #216 till the server validates the date provided is valid
+            for count in range(0, len(result["Items"])):
+                if result["Items"][count].get('ProductionYear'):
```

Comment:
Mixing single and double quotes.  tsk tsk

I prefer single in most cases, though the addon is already mixing them all over the place.  We just need to decide on one and be consistent with it.

Reply:
Personally, I tend to use single quote for dictionary indexing, and double quote for "normal" strings. No particular reason why, it's just how I've been doing it over the years..